 
  Serverless Error ---------------------------------------
 
  Serverless plugin "serverless-step-functions" not found. Make sure it's installed and listed in the "plugins" section of your serverless config file.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          linux
     Node Version:              8.10.0
     Framework Version:         1.54.0
     Plugin Version:            3.1.2
     SDK Version:               2.1.2
     Components Core Version:   1.1.1
     Components CLI Version:    1.2.3
 
 
  Serverless Error ---------------------------------------
 
  Serverless plugin "serverless-step-functions" not found. Make sure it's installed and listed in the "plugins" section of your serverless config file.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          linux
     Node Version:              8.10.0
     Framework Version:         1.54.0
     Plugin Version:            3.1.2
     SDK Version:               2.1.2
     Components Core Version:   1.1.1
     Components CLI Version:    1.2.3
 
 
  Serverless Error ---------------------------------------
 
  Serverless plugin "serverless-step-functions" not found. Make sure it's installed and listed in the "plugins" section of your serverless config file.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          linux
     Node Version:              8.10.0
     Framework Version:         1.54.0
     Plugin Version:            3.1.2
     SDK Version:               2.1.2
     Components Core Version:   1.1.1
     Components CLI Version:    1.2.3
 

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T12_11_51_022Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  13 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T12_15_53_938Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  13 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T12_19_56_770Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (23ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T12_23_59_672Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T12_28_02_512Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  13 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T12_32_05_344Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  13 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T12_36_08_145Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  13 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T12_40_11_044Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (23ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T12_44_13_944Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  13 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T12_48_16_733Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  13 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T12_52_19_581Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (23ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T12_56_22_475Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (23ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_00_25_374Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  13 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_04_28_171Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_08_31_011Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_12_33_919Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_16_36_811Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  13 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_20_39_741Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (23ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_24_42_620Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  13 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_28_45_434Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  13 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_32_48_292Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (23ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_36_51_099Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (23ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_40_54_001Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (22ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_44_56_806Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [/api]
Done!

  Article
**** Call ## 0 ## Duration:  14 (ERR)
    1) "before all" hook

  0 passing (23ms)
  1 failing

  1) Article
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:80
      at Object._errnoException (util.js:1022:11)
      at _exceptionWithHostPort (util.js:1044:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)




npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-11-01T13_48_59_685Z-debug.log
module.js:549
    throw err;
    ^

Error: Cannot find module 'watchtower-anaysis-scripts'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/ubuntu/realworld-dynamodb-lambda/get-wt-times.js:1:79)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
 
  Serverless Error ---------------------------------------
 
  Serverless plugin "serverless-step-functions" not found. Make sure it's installed and listed in the "plugins" section of your serverless config file.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          linux
     Node Version:              8.10.0
     Framework Version:         1.54.0
     Plugin Version:            3.1.2
     SDK Version:               2.1.2
     Components Core Version:   1.1.1
     Components CLI Version:    1.2.3
 
